packages/rich-text/src/component/index.native.js

```diff
@@ -679,8 +698,8 @@ export class RichText extends Component {
 			value = '';
 		}
 		// On android if content is empty we need to send no content or else the placeholder will not show.
-		if ( ! this.isIOS && value === '' ) {
-			return value;
+		if ( ! this.isIOS && ( value === '' || value === '<p></p>' ) ) {
```

Comment:
In here instead of doing `value === <p></p>` you should write `value === '<'+tagname+'></'+tagname'>'`

Reply:
I understand the thought here, but there are a few issues. First off to catch this issue for Quote/Pullquote and List Block we would want to use not `tagName`, but `this.multilineTag` which seems to be the tag that is staying in `value` that is preventing us from seeing the placeholder. Even if we do switch this to target the general case of a remaining `this.multilineTag`, removing this for the List Block seems to create issues that are quite unique from the Quote and Pullquote block (basically the ListBlock no longer acts like a list block), different enough to justify a separate fix for List Block in my opinion.